### PR TITLE
Update codemirror.php

### PR DESCRIPTION
**Pull request by wldev1**
Removed unsued lines

### CODE REVIEW

**Comment by brianteeman** [comment]
There are changes in this PR that are more than just "removing unused lines"

On 6 March 2015 at 11:50, wldev1 <notifications@github.com> wrote:

> Removed unsued lines
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/joomla/joomla-cms/pull/6334
> Commit Summary
>
>    - Update codemirror.php
>
> File Changes
>
>    - *M* plugins/editors/codemirror/codemirror.php
>    <https://github.com/joomla/joomla-cms/pull/6334/files#diff-0> (108)
>
> Patch Links:
>
>    - https://github.com/joomla/joomla-cms/pull/6334.patch
>    - https://github.com/joomla/joomla-cms/pull/6334.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/6334>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

File: plugins/editors/codemirror/codemirror.php [hunk 1]
```diff
@@ -27,60 +19,60 @@ class PlgEditorCodemirror extends JPlugin
...
 
 	/**
 	 * Mapping of syntax to CodeMirror modes.
-	 *
 	 * @var array
 	 */
 	protected $modeAlias = array(
-			'html' => 'htmlmixed',
-			'ini'  => 'properties'
+			var $html = 'htmlmixed',
```
Comment (Fedik) [inline plugins/editors/codemirror/codemirror.php]: what is it? :smile:

**Comment by Fedik** [comment]
I think this pull is a stupid joke :smile:

**Comment by brianteeman** [comment]
I am going to close this PR as it is clearly not doing what it says. If the PR or description are updated to correctly match it can always be reopened

**Pull request closed without merging**